force-app/main/default/classes/UTIL_Query.cls

```diff
@@ -110,6 +121,20 @@ public with sharing class UTIL_Query {
         return this;
     }
 
+    /**
+     * @description Registers an ORDER BY field only if that field is unencrypted and sortable.
+     * @param field A reference to the field being sorted on
+     * @param orderByClause The order by clause to add to the query
+     *
+     * @return UTIL_Query The instance of the Query builder
+     */
+    public UTIL_Query withOrderByEncryptable(Schema.SObjectField field, String orderByClause) {
```

Comment:
I don't think we need the "encryptable" piece of the method name here. We could just have withOrderBy overloaded since the signature is different and we can just know to use this signature if we want to check if the field is sortable first since there are other instances where a field could not be sortable other than encryption such as long text area.